ui/lib/components/CardTable/index.module.less

```diff
@@ -1,6 +1,6 @@
-.cardTable {
+.card-table {
   :global {
-    .ms-DetailsRow {
+    .ms-details-row {
```

Comment:
Not sure whether ".ms-DetailsRow" is a class for the Microsoft Fluent DetailsList component, need to check it.